slackbot-background/handlers.go

```diff
@@ -50,8 +50,13 @@ func handleWhitelistCmd(ctx context.Context, cmd common.SlashCommandData, db *co
 		return msg, err
 	}
 
+	err = deleteIpFromIprepd(ip.String())
```

Comment:
Do we verify the format of this string to ensure it is actually an address? I didn't see that anywhere but might have missed it.

Reply:
Yeah, this is done within the parse function:

https://github.com/mozilla-services/foxsec-pipeline-contrib/blob/a99ae9291b222c89fc1175b573d1be40fc1560b0/slackbot-background/utils.go#L40-L48

Reply:
👍